### PR TITLE
56 New Icons + Complete Update of Existing Icons

### PR DESCRIPTION
**Pull request by RobertHarnett**
## TL;DR: 
- Added 56 New Icons
- Updated any out of date icons (eg; Adobe).
- Remade some icons that looked low quality (eg: onshape), I admit I don't know how noticeable this is when scaled down but it allows for greater flexibility in the future if you ever decide to alter icon scaling.)
- Remade ones that were scaled incorrectly  to the 128x128 size (eg patreon).
- Note there's a fair few where they look nearly identical to existing icons, on some, the logo's have changed a tiny little bit such as a small colour change, on others they are essentially identical. At the stage where I was comparing any of the existing icons to remade icons I had done, I had basically done the hard part already and would only need to save as png, so If I remade one and it turned out to be the same I decided to still export and replace the existing icon because it looked like there was no Anti-Aliasing on the existing icons. 'onshape.png' is the clearest example of this.
- Deleted icons that are no longer needed, such as unnecessary dark mode versions, icons of services that don't exist, 'Version2' of any icons as these feel redundant, and icons of services that use a account under a different name eg: 'Yandex mail' falls under 'Yandex'.
- Updated missing_icons.txt

### New Icons:
- Addiko Bank
- Advcash (+ Dark Mode)
- Automate.io
- Avanza
- Betfair
- BigMind
- Bitgo
- Brightbox
- Cake defi
- Call of Duty (+ Dark Mode)
- Cisco
- Clearscore (+ Dark Mode)
- Clubhouse (+ Dark Mode)
- Community America
- Crashplan
- Datto
- Degiro
- Ecobee[^Ecobee]
- Fastly
- Files.com
- Gaijin
- Google Cloud
- Hey
- Hodlenaut
- IBM
- ID.me
- idrive.com
- Intuit[^Intuit]
- Ionos
- Jovia
- Koofr
- Luno (+ Dark Mode)
- MacStadium (+ Dark Mode)
- Make
- Mercury (+ Dark Mode)
- Meta
- Morgan Stanley (+ Dark Mode) (Easily the most boring logo I've ever seen, not much I could do here)
- Nordlayer
- Nutstore
- Onehub
- Optimizely
- Oracle
- Pusher (+ Dark Mode)
- Rewind
- Runescape (+ Dark Mode)
- SEFCU
- Sony (+ Dark Mode)
- Standard Notes
- Strato
- Sync
- Trading 212
- Ubuntu One
- VMware
- VR Chat
- Whois
- Xero

### Updated Icons:
- 1and1
- 1password
- 3cx (Slightly darker grey)
- Adobe
- Airship
- Allegro
- Altaro
- Amazon
- Appveyor
- Arduino
- Asana
- Ascendex
- Atlassian
- Autodesk (+ Dark Mode)
- AWS (+ Dark Mode)
- Azure
- Basecamp
- Best Buy
- Binance
- Bitcoin
- Bitdefender
- Bitfinex
- Bitflyer
- Bitstamp (+ Dark Mode)
- Bittrex
- Blizzard
- Blockchain
- Bluehost
- Box
- Buildkite
- BunnyCDN
- Codeberg (+ Dark Mode)
- Codeship (+ Dark Mode)
- Coinjar (+ Dark Mode)
- Confluence
- Contabo
- Control D (+ Dark Mode)
- Crypto.com (+ Dark Mode)
- Dashlane (+ Dark Mode)
- Deutschebahn
- Direct Admin
- Discogs (+ Dark Mode)
- Discord
- Docker 
- Dreamhost
- Dropbox
- Electronic Arts
- Envato
- Evernote
- Facebook
- Figma
- Filen (+ Dark Mode)
- Floatplane
- fmfw
- FTX
- Gemini
- Gitlab
- Gmail
- GoDaddy
- Google
- Google Drive
- Google Fi
- Google Fibre
- Google Pay
- Google Play
- Grammarly
- Hellosign
- Heroku
- Hitbtc
- Home Assistant
- Hotbit
- Humble Bundle[^HumbleBundle]
- Huobi
- Instagram
- Intigriti[^Intigriti]
- InVision
- IP Phone Forum
- Itglue
- Jagex
- Jelastic
- Jetbrains (+ Dark Mode)
- Jira
- Jottacloud
- Kaspersky
- Kraken
- Leaseweb
- Linkedin
- Linode
- Litebit
- Local Cryptos
- Local Monero
- Login.gov
- Maicoin
- Mailbox.org
- Mailchimp (+ Dark Mode)
- Mail.com
- Mailfence
- Mailgun
- Mailo
- Mapbox (+ Dark Mode)
- Mastodon
- Matomo
- Mega
- Microsoft
- Microsoft Todo
- Migros
- Mintos
- Monday.com
- Moneyforward
- Moneytree
- Moqups
- Myheritage
- Namecheap
- Name.com (+ Dark Mode)
- Netcup
- Nintendo[^Nintendo]
- Nuget
- Nulab
- NURI
- Onedrive
- Onshape
- Origin
- Outlook
- ovh
- Parsec
- Patreon
- Paykassa
- Paypal
- Paysafecard
- Plex
- Plurk
- Poloniex_Dark
- Private Internet Access
- Proxmox_dark
- Pushover
- Pypi
- Qnap
- Rackspace
- RealVNC
- Robinhood
- Roblox (+ Dark Mode)
- Roboform
- RocketbeansTV
- Salesforce
- Samsung[^Samsung]
- Scaleway
- Shortio
- Simplelogin
- Skype
- Slack
- Snapchat
- Socketlabs
- Solarwinds
- Sproutsocial
- Square
- Squarespace
- Stackpath (+ Dark Mode)
- Statuspage
- Steam (+ Dark Mode)
- Stormgain
- Surfshark
- Synology
- Team Viewer
- Telnyx
- Ting
- T-Mobile
- Tokopedia
- Tor Guard (+ Dark Mode)
- Trello
- Tresorit
- Tutanota
- Twilio
- Twitch
- Twitter
- Ultimaker
- United Domains
- Unlock Base
- Unstoppable Domains
- Updraft
- UpTimeRobot
- Visual Studio Online
- VK
- Voyager
- Vultr
- War Gaming
- Web.de
- Whale Fin
- wordfence
- Wyze (Again... Updated colour and text weight)
- ynab
- YoYo Games
- Zoho
- Zonda (+ Dark Mode)
- Zoom
- Zyxel (+ Dark)

### Deleted Icons:
Some of these have dark mode versions that are either not necessary because they are already legible in both light and dark mode or because they have an updated icon where this is now the case (I previewed them in my icon designer software with the same background colours used in the app to make sure they are clear to see).
- 34sp (Dark Mode)
- Adobe (Dark Mode)
- BasicAttentionToken is a crypto currency by Brave, it's not a service on its own and does not require a icon.
- Best Buy (Dark Mode)
- Box (Dark Mode)
- Crypton I can't find this service anywhere? Even reverse image searched the current icon and it seems it doesn't exist?
- Electronic Arts (Dark Mode)
- Google Cloud Platform (Now called 'Google Cloud')
- Google Pay (Dark Mode)
- Gildwars2 Uses Arenanet to login which already has a account.
- Hangouts (The service no longer exists)
- Integromat has rebranded into 'Make' have added a new icon for it.
- Integrity was misnamed, has simply been named 'Intigriti', icon still exists.
- Jetbrains2 (Removed the logo with the coloured line as it makes the logo really small, using just the square on its own is still within the brand guidelines.
- Patreon (Dark Mode)
- Netcup (Dark Mode)
- Oculus now exists under the 'Meta' name. I have deleted the old logo and created one for Meta.
- Office365 has rebranded as Microsoft365, I still wanted to keep a icon for office for user choice so have updated the icon and named it simply 'Office'
- Samsung[^Samsung]
- 'simpletax.co' looks 100% like a scam or very old proprietary software that almost no one uses. There's several hundred other services I would prefer we make an icon for first rather then this sketchy looking site. What caused this icon to be added? Was it a legitimate user request? I decided to remove it because I question it's legitimacy, while you could argue that's not for the job of someone submitting icons, I really hate this icon and don't want the icon selector to be clogged full of spammy random sites.
- Smashcast no longer exists.
- Starling Bank Developer (Renamed file to simply 'starlingbank')
- Statusmail (Is this an actual service, it seems to just be part of 'statuspage' which already has an icon.
- Synology (Dark Mode)
- Uplay, it's rebranded to 'Ubisoft Connect' and uses a ubisoft account which already has an icon.
- Yandex Mail (Uses a Yandex account which can be for multiple services, I have added a icon for 'Yandex'

### Updated 'missing_icons.txt':
- Removed names of added icons
- 'NordVPN Teams' has rebranded as Nord Layer, icon added.
- 'Bugzilla@Mozilla' uses the 'Mozilla' logo which already exists.
- 'Hover.com' is already added.
- TurboTax, Quickbooks, & Mint all use a shared 'Intuit' account
- removed 'Ukraine' because... well idk what it is other then a country, and I don't think an entire country has 2FA. Well I suppose it probably in a way does have two factor authentication to enter it right now but that's a differant thing entirely.
- Removed 'Samsung Smartthings' as far as im aware it uses a Samsung account.
- 'Remote Desktop Manager' & 'Online' could be any number of services, Im removing them as I don't actually know what the services are.
- 'Launchpad' uses Ubuntu One login, icon added.
- join.me uses 'LogMeIn' which already has an icon.

[^Ecobee]: Ecobee's Logo is rather thin text so this icon is based off of their app icon to improve legibility, funnily enough they seem to realise this problem themselves as they use the same solution for their favicon on their website.

[^Intigriti]: Previously misnamed as 'Integrity', not deleted, just renamed.

[^Intuit]: Their logo is just text which is hard to see so again I've based this icon off of their favicon.

[^HumbleBundle]: Both the old 'H' icon and the bag icon are within official brand guidelines and can both be used, I decided to change to the bag to reduce the use of typography in the icon set, this is a personal preference so let me know if you prefer to use the 'H'.

[^Nintendo]: Even on my Pixel 6 Pro which is both on the higher end of screen size and device PPI the current Nintendo logo is very small and hard to read, here I copied what Nintendo have done on their own website it regards to their favicon. This is again a personal preference to reduce the dependence on type heavy icons. Let me know if I should revert this change.

[^Samsung]: Samsung no longer use the rounded blue logo, so I have deleted it and made the 'samsung2' icon the primary and only Samsung logo.

### CODE REVIEW

**Comment by jamie-mh** [comment]
Hi @RobertHarnett ,

Thanks for taking the time to cleanup and update the icon set. I really appreciate it.

I totally agree with most of the changes.

Just a few small comments:

- The Moneytree icon is for the wrong Moneytree, this is the right one: https://getmoneytree.com/jp/home
- I think the Steam icon looks better in colour than black and white.
- The reason why the Nintendo logo is like that is because I don't know if it's a good idea to use Mario. Nintendo is a litigious company and this might be a problem.

I will update the icon guidelines regarding niche services, no point polluting the list with icons nobody uses. Custom icons can be used in that case.

**Comment by RobertHarnett** [comment]
Taken your comments onboard and changed the relevant icons.

In regards to limiting and choosing the icons to create I wanted to make more changes but didn't want this pull request to be 'controversial' and just wanted it to be a simple refresh. To get the ball going in that area I've opened an issue #449 to discuss it and allow this pull request to be merged.
